RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+13.8
+-----
+* Block editor: Images clickable for fullscreen preview
```

Comment:
Small detail, `clickable` might not be the best word?
What about something like:
```suggestion
* Block editor: Images from Image Block can now be previewed full screen by tapping on them.
```

Reply:
That does sound more clear. Updated, thanks!